### ISSUE TITLE
ListOfStyles is not iterable

### ISSUE THREAD

**Issue by matthiaskoenig** [fbergmann/libSEDML#104]
ListOfs are all iterable in Python, but the ListOfStyles misses the respective part in pyswig:
```
for sed_style in sed_doc.getListOfStyles():
    self.styles[sed_style.getId()] = self.parse_style(sed_style)
```

```
File "/home/mkoenig/git/sbmlsim/src/sbmlsim/combine/sedml/parser.py", line 203, in __init__
    for sed_style in sed_doc.getListOfStyles():
TypeError: 'SwigPyObject' object is not iterable
```

Workaround is to just iterate explicitly using the index until this is fixed.
```
        for k in range(sed_doc.getNumStyles()):
            sed_style: libsedml.SedStyle = sed_doc.getStyle(k)
            self.styles[sed_style.getId()] = self.parse_style(sed_style)
```

**Comment by fbergmann** [comment]
this is fixed in 2.0.17 (now on pypi)

**Comment by matthiaskoenig** [comment]
Works. Thanks.
